### PR TITLE
MW-720, adds the network pie chart feature.

### PR DESCRIPTION
**Pull request by vladcipariu91**

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
         <option name="modules">
           <set>
             <option value="$PROJECT_DIR$" />
+            <option value="$PROJECT_DIR$/app" />
```
Comment (tomermandelfw) [inline .idea/gradle.xml]: I guess this should be removed?

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 *.iml
 .gradle
 /local.properties
+/.idea
```
Comment (igorcferreira) [inline .gitignore]: @vladcipariu91 @tomermandelfw The hole `.idea` folder should not be removed. This folder has useful files for the project, like codestyles and similar. Please, revert this change and add any relevant exclusion like the chaches and libraries bellow.

**Review by igorcferreira — DISMISSED** [review]

File: .gitignore (hunk 2 above)
Comment (vladcipariu91) [reply .gitignore]: ok

File: .idea/gradle.xml (hunk 1 above)
Comment (vladcipariu91) [reply .idea/gradle.xml]: @tomermandelfw done

File: .gitignore (hunk 2 above)
Comment (vladcipariu91) [reply .gitignore]: @igorcferreira done.

**Review by igorcferreira — APPROVED** [review]
